File: src/musicGame/util/MapEntry.java
package musicGame.util;

public class MapEntry<K, V> {
  private K key;
  private V value;

  private MapEntry(K key, V value) {
    this.key = key;
    this.value = value;
  }

  public static <K, V> MapEntry<K, V> entry(K key, V value) {
    return new MapEntry<K, V>(key, value);
  }

  public K getKey() {
    return key;
  }

  public V getValue() {
    return value;
  }
}

File: java/believe/util/MapEntry.kt
package believe.util

class MapEntry<K, V> private constructor(val key: K, val value: V) {
    companion object {
        @JvmStatic
        fun <K, V> entry(key: K, value: V): MapEntry<K, V> {
            return MapEntry(key, value)
        }
    }
}